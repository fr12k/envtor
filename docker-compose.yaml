version: "3"
services:
  "1":
    image: bash:latest
    command: bash -c "printenv | sort"
    environment:
      - ALL_ENV_VARS
      - GO_ENV=production
  "2":
    image: bash:latest
    command: bash -c "printenv | sort"
    environment:
      - ENV=test
      - ALL_ENV_VARS
      - GO_ENV=production
    ports:
      - "50051:50051"
    networks:
      - mynet
    volumes:
      - storage:/app

networks:
  mynet:
    driver: bridge

volumes:
  storage:
