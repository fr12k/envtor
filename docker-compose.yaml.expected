networks:
    mynet:
        driver: bridge
services:
    "1":
        command: sh -c "env | sort"
        environment:
            - ENVIRONMENT1=hello
            - ENVIRONMENT2=world
            - GO_ENV=production
        image: alpine
    "2":
        command: sh -c "env | sort"
        environment:
            - ENV=test
            - ENVIRONMENT1=hello
            - ENVIRONMENT2=world
            - GO_ENV=production
        image: alpine
        networks:
            - mynet
        ports:
            - 50051:50051
        volumes:
            - storage:/app
version: "3"
volumes:
    storage: null
